foundry.toml

```diff
@@ -18,4 +18,22 @@ src = 'src'
 test = 'src'
 via_ir = true
 
-# See more config options https://github.com/foundry-rs/foundry/tree/master/config
+[rpc_endpoints]
+mainnet = "https://eth-mainnet.g.alchemy.com/v2/${MAINNET_ALCHEMY_KEY}"
```

Comment:
fun fact alchemy allows access for any network from one API key so this can be the same. fine as-is though